sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/meta/provider/bigquery/BeamBigQueryTable.java

```diff
@@ -34,6 +36,8 @@
 /**
  * {@code BeamBigQueryTable} represent a BigQuery table as a target. This provider does not
  * currently support being a source.
+ *
+ * <p>New BigQuery table is created if it does not exist.
```

Comment:
... is created when written ...

Is that accurate? I think just doing a `CREATE TABLE` will not cause it to be created?

And what will the schema be? Does it support `CREATE TABLE` with no schema, and pulls it down if we have the credentials? We should document that a little here but mostly in the SQL reference docs.

Reply:
Removed the automatic creation for now. `TestBigQuery` rule does it, to avoid cleaning up something we didn't create